### PR TITLE
deployment config for svobs website instance

### PR DESCRIPTION
**Pull request by shifucun**
* Sync mixer to use the "svobs_mode" flag
* Add deployment config for svobs instance
* Remove unused build/ci/autopush.yaml file
* Make two deployment branches, "popobs" and "svobs" to use corresponding schema mapping and storage version files

---
This change is [Reviewable](https://reviewable.io/reviews/datacommonsorg/website/842)

### CODE REVIEW

File: deploy/overlays/svobs/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+patchesStrategicMerge:
+  - |-
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: website-app
+    spec:
+      replicas: 6
+
+patchesJson6902:
```
Comment (beets) [inline deploy/overlays/svobs/kustomization.yaml:48]: could you describe what is this patch for? i'm curious where "json6902" came from

**Review by beets — COMMENTED** [review]
thanks bo!

would you mind listing steps you took to generate these files (or if you had manually configured them)? just so there's a trail for someone else to add a deployment in the future.

**Review by pradh — APPROVED** [review]
+1 :)

> *[Reviewable](https://reviewable.io/reviews/datacommonsorg/website/842#-:-MVN58ua9-xhCN3GsLEV:b-8o2if0)* status: 0 of 18 files reviewed, 3 unresolved discussions (waiting on @shifucun)

___
*[deploy/svobs/kustomization.yaml, line 23 at r1](https://reviewable.io/reviews/datacommonsorg/website/842#-MVN4dqkBrvPu4-tsZ_U:-MVN4dqkBrvPu4-tsZ_V:b-7rox9f) ([raw file](https://github.com/datacommonsorg/website/blob/992922265363b2e4a49f19f745c4a6cd288ae9ef/deploy/svobs/kustomization.yaml#L23)):*
> ```
> statvar
> ```

Should we consistently use svobs everywhere to refer to the new mode?
___
*[gke/mci.yaml.tpl, line 21 at r1](https://reviewable.io/reviews/datacommonsorg/website/842#-MVN4kSL24YeQK2tre9a:-MVN4kSL24YeQK2tre9b:b-tuf3jv) ([raw file](https://github.com/datacommonsorg/website/blob/992922265363b2e4a49f19f745c4a6cd288ae9ef/gke/mci.yaml.tpl#L21)):*
> ```
> multi-domain
> ```

A comment for what this is ?

**Review by shifucun — COMMENTED** [review]
Added a guide for bringing up new instance

> *[Reviewable](https://reviewable.io/reviews/datacommonsorg/website/842#-:-MVO8shYAEDl2g9HU_x5:b-dvcx93)* status: 0 of 18 files reviewed, 3 unresolved discussions (waiting on @beets and @pradh)

___
*[deploy/overlays/svobs/kustomization.yaml, line 48 at r1](https://reviewable.io/reviews/datacommonsorg/website/842#-MVN4NmZTe_Giun7ELtb-r1-48:-MVO84Is8Q_1SspvPNuB:bemx3rf) ([raw file](https://github.com/datacommonsorg/website/blob/992922265363b2e4a49f19f745c4a6cd288ae9ef/deploy/overlays/svobs/kustomization.yaml#L48)):*
<details><summary><i>Previously, beets wrote…</i></summary><blockquote>

could you describe what is this patch for? i'm curious where "json6902" came from
</blockquote></details>

Added comments, it's a way of patching the yaml file, good for modifying list items: https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/patchesjson6902/
___
*[deploy/svobs/kustomization.yaml, line 23 at r1](https://reviewable.io/reviews/datacommonsorg/website/842#-MVN4dqkBrvPu4-tsZ_U:-MVO8pnl2Mr86FLZdB9X:bb6un1j) ([raw file](https://github.com/datacommonsorg/website/blob/992922265363b2e4a49f19f745c4a6cd288ae9ef/deploy/svobs/kustomization.yaml#L23)):*
<details><summary><i>Previously, pradh (Prashanth R) wrote…</i></summary><blockquote>

> ```
> statvar
> ```

Should we consistently use svobs everywhere to refer to the new mode?
</blockquote></details>

Done
___
*[gke/mci.yaml.tpl, line 21 at r1](https://reviewable.io/reviews/datacommonsorg/website/842#-MVN4kSL24YeQK2tre9a:-MVO8rWf4q52kyC60WLX:b-896fix) ([raw file](https://github.com/datacommonsorg/website/blob/992922265363b2e4a49f19f745c4a6cd288ae9ef/gke/mci.yaml.tpl#L21)):*
<details><summary><i>Previously, pradh (Prashanth R) wrote…</i></summary><blockquote>

> ```
> multi-domain
> ```

A comment for what this is ?
</blockquote></details>

Done.

**Pull request merged**